domain/src/main/java/com/thoughtworks/go/domain/materials/git/GitMaterialUpdater.java

```diff
@@ -140,7 +140,7 @@ private BuildCommand cleanWorkingDir(String workDir) {
 
     private BuildCommand isRepoUrlChanged(String workDir) {
         return test("-neq",
-                material.getUrlArgument().forCommandline(),
+                material.getUrlArgument().rawUrl(),
```

Comment:
This should be changed back to `forCommandline()`

Reply:
done